uktrade/data-hub-frontend
# Fix/Display calculated GVA under the correct field

## Description of change
When editing an FDI investment project, the GVA calculation and messaging currently displays under the 'Capital expenditure' field. This is the correct behaviour for a capital intensive project but in the case of a labour intensive project it should instead be displayed under the 'Number of new jobs' field. This PR corrects this behaviour.

## Test instructions
1. 
- Create a capital intensive project (e.g. select investment type "FDI" and sector "Advanced Engineering")
- Select "Edit Value"
- Select "Yes" for the "Can client provide capital expenditure value?" field
- When no figure has been added, the "Gross value added (GVA)" field should be displayed under the "Capital expenditure field" and will display a message prompting the user to enter a value and hit save
- The "Gross value added (GVA)" field should not appear under the "Number of new jobs" field

<img width="666" alt="Screenshot 2024-05-21 at 15 09 45" src="https://github.com/uktrade/data-hub-frontend/assets/74198488/40129d49-f2e1-4dbd-9b78-0b403d27966c">

- Enter a value into the "Capital expenditure value" field
- Add a value into the "Number of new jobs" field
- Click save at the bottom of the from
- Return to the "Edit Value" form
- The "Gross value added (GVA)" field should now display the calculated GVA value under the "Capital expenditure value" field
- The "Gross value added (GVA)" field should not appear under the "Number of new jobs" field

<img width="666" alt="Screenshot 2024-05-21 at 15 38 37" src="https://github.com/uktrade/data-hub-frontend/assets/74198488/f90f9ddd-2296-4ac0-a2ec-270b36ecbc83">


2. 
- Create a labour intensive project (e.g. select investment type "FDI" and sector "Construction")
- Select "Edit Value"
- When no figure has been added, the "Gross value added (GVA)" field should be displayed under the "Number of new jobs" field and will display a message prompting the user to enter a value and hit save.
- Select "Yes" for the "Can client provide capital expenditure value?" field
- The "Gross value added (GVA)" field should not appear under the "Capital expenditure value" field

<img width="666" alt="Screenshot 2024-05-21 at 15 10 55" src="https://github.com/uktrade/data-hub-frontend/assets/74198488/d8a24f29-5759-4593-afac-a1bb7cf9e20a">

- Enter a value into the "Number of new jobs" field
- Select "Yes" for the "Can client provide capital expenditure value?" field
- Enter a value into the "Capital expenditure value" field
- Click save at the bottom of the from
- Return to the "Edit Value" form
- The "Gross value added (GVA)" field should now display the calculated GVA value under the "Number of new jobs" field
- The "Gross value added (GVA)" field should not appear under the "Capital expenditure value" field
- 
<img width="666" alt="Screenshot 2024-05-21 at 15 40 09" src="https://github.com/uktrade/data-hub-frontend/assets/74198488/6a581468-b97d-4331-8f93-79ff485a174b">

## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
